static/app/components/breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import {Fragment} from 'react';
+
+import {Breadcrumbs} from 'sentry/components/breadcrumbs';
+import * as Storybook from 'sentry/stories';
+
+export default Storybook.story('Breadcrumbs', story => {
+  story('Basics', () => (
+    <Fragment>
+      <p>
+        <Storybook.JSXNode name="Breadcrumbs" /> displays a navigation path with clickable
+        links. By default, the last item is not clickable to avoid linking to the current
+        page.
+      </p>
+      <Storybook.SizingWindow display="block">
+        <Breadcrumbs
+          crumbs={[
+            {label: 'Organization', to: '/organizations/sentry/'},
+            {label: 'Projects', to: '/organizations/sentry/projects/'},
+            {label: 'Project Settings', to: '/settings/projects/javascript/'},
+            {label: 'General', to: null},
+          ]}
+        />
+      </Storybook.SizingWindow>
+    </Fragment>
+  ));
+
+  story('With Last Item Linked', () => (
+    <Fragment>
+      <p>
+        Set <Storybook.JSXProperty name="linkLastItem" value /> to make the last
+        breadcrumb clickable.
+      </p>
+      <Storybook.SizingWindow display="block">
+        <Breadcrumbs
+          linkLastItem
+          crumbs={[
+            {label: 'Organization', to: '/organizations/sentry/'},
+            {label: 'Projects', to: '/organizations/sentry/projects/'},
+            {label: 'All Projects', to: '/organizations/sentry/projects/all/'},
```

Comment:
```suggestion
            {label: 'All Projects', to: '/organizations/sentry/projects/'},
```

i think the 'all' is not needed, it leads to an error